### PR TITLE
Update nginx.md

### PR DESCRIPTION
**Pull request by Neosporin10**
I wanted to add some stuff to this great tutorial!

### CODE REVIEW

File: remote-access/web-server/nginx.md [hunk 1]
```diff
@@ -130,3 +130,76 @@ Add some dynamic PHP content by replacing the current content:
...
+What next! - I'll be adding a guide for using Lets Encrypt to add ssl to the Raspberry Pi nginx server.
+
+In this tutorial, I’ll guide you through the process of installing let’s encrypt SSL certificates on your nginx powered website. By the end of the tutorial, we’ll have done the following:
+
+Installed the let’s encrypt service
+Generate a free let’s encrypt SSL certificate
+Install free SSL certificate in nginx to secure your site
+Secure site with SSL
+
+Throughout this tutorial, I’ll be installing and configuring in Ubuntu 16.04, but the instructions will also work fine with Debian. I haven’t tested but there’s no reason why this tutorial won’t work to install SSL to your nginx site on Raspbian for a Raspberry Pi.
```
Comment (lurch) [inline remote-access/web-server/nginx.md:143]: "I’ll be installing and configuring in Ubuntu 16.04..." - the documentation in this repo is written first and foremost for Raspbian, as that's the only officially-supported Raspberry Pi OS.

**Comment by lurch** [comment]
The usage of "I ..." and "we ..." is *very* different to the writing-style used in the rest of the documentation :-/

**Comment by JamesH65** [comment]
Agree about the writing style. it will need to be changed to use the third person (or whatever we call it!). And will also need to be tested on Raspbian, as @lurch says, this docs repo is specifically for Raspbian.

**Comment by Neosporin10** [comment]
Yes I should take that back but would be cool to add a way to add ssl to
the doc! This way doesn't work shouldn't have put in that commit

**Comment by Neosporin10** [comment]
I'm working on my pi nginx server! Https:www.mikeyraptor.dynu.net

**Comment by JamesH65** [comment]
Unless the requests above are likely to be implemented, I'm going to close this PR. At the moment it's not suitable for our docs repo.

**Comment by JamesH65** [comment]
OK, so no response from the OP, so closing. Could be reopened if above criteria are met.

**Pull request closed without merging**